about.html

```diff
@@ -3,24 +3,49 @@ <h2 id="rules">Rules <a href="#rules">¶</a></h2>
 <em>"Be excellent to each other"</em> is easier said than done, and means different things to different people.
 </p>
 <p>
-The following rules are a (non-exhaustive) list of behaviours that may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators.
+The following rules are a (non-exhaustive) list of behaviours that may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators, as described in our <a href="/terms">Terms</a>.
 </p>
 <p>
 Please report behaviour that bothers you. We will keep your report confidential.
 </p>
 <ul>
-<li>We do not tolerate harrassment, stalking, and doxxing.</li>
-<li>We do not tolerate racism, sexism, verbal abuse, discrimination against gender and sexual minorities, or the advocation of any of these.</li>
+<li>We do not tolerate discriminatory behaviour and content promoting or advocating the oppression of members of marginalised groups. These groups may be characterised by any of the following (though this list is naturally incomplete):
+  <ul>
+    <li>ethnicity</li>
+    <li>gender identity or expression</li>
+    <li>sexual identity or expression</li>
+    <li>physical characteristics or age</li>
+    <li>disability or illness</li>
+    <li>nationality, residency, citizen status</li>
+    <li>wealth or education</li>
+    <li>faith, or lack thereof</li>
```

Comment:
```diff
-faith, or lack thereof
+religious beliefs, agnosticism or atheism
```

Reply:
"religious affiliation or lack thereof" is a good mix of the two, I think.

Reply:
Would you say "lack of faith" or "lack of religious belief" is judgemental? Somehow to me this reads like "over here are people *with* religious beliefs, those are the normal people. Over there are some that lack sufficient belief". I know this is not the intention, this is the cartoonish distorted version to illustrate what I tried to avoid with `religious beliefs, agnosticism or atheism`.

Reply:
Ahh, I see where you're coming from. I interpreted your change to be mostly about the change from "faith" to "religious beliefs" and didn't catch the implication of a "lack". I'll add "agnosticism or atheism" instead, thank you for the suggestion and explanation!